lib/facade/FileClient.ts

```diff
@@ -36,9 +36,7 @@ export class FileClient {
     source: ReadableStream | Buffer | string | ArrayBuffer,
```

Comment:
We still should probably create NodeStream interface and add here at least for readability (and add comment why we not using node stream as type

Reply:
I've added new "InputSource" type for this. 
I suppose it's neither Stream actually, looks more like input data source
Let me know if i'm wrong